src/components/Sum.jsx

```diff
@@ -1 +1,27 @@
-// Don't forget to add default values an check types of props
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Sum.scss';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
+  <p className="addition">
+    {'Sum of '}
+    {a}
+    {' and '}
+    {b}
+    {' is '}
+    {a + b}
```

Comment:
```suggestion
    {`Sum of ${a} and ${b} is ...${a + b}`}
```
Actually it could be written like this